cmd/antrea-agent/options.go

```diff
@@ -307,13 +307,17 @@ func (o *Options) validateAntreaIPAMConfig() error {
 }
 
 func (o *Options) validateMulticlusterConfig(encapMode config.TrafficEncapModeType) error {
-	if !o.config.Multicluster.Enable {
-		o.config.Multicluster.EnableStretchedNetworkPolicy = false
+	if !o.config.Multicluster.EnableGateway && !o.config.Multicluster.EnableStretchedNetworkPolicy {
 		return nil
 	}
 
 	if !features.DefaultFeatureGate.Enabled(features.Multicluster) {
-		return fmt.Errorf("Multicluster can only be enabled when Multicluster feature gate is enabled")
+		klog.InfoS("Multicluster feature gate is disabled. Multi-cluster options are ignored")
```

Comment:
@GraysonWu : I found other features (e.g. NodePortLocal) ignore config options when the feature gate is not enabled, which makes to me, and so made this change.